### PR TITLE
Add work batch and user task endpoints

### PR DESCRIPTION
**Pull request by withrocks**
This includes various changes mainly to provide an aggregate endpoint for querying for user tasks.

### CODE REVIEW

File: src/clims/models/camunda.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# Models for camunda. None of these is created by CLIMS, but we require them here for querying them more easily
+# when the API isn't enough
+
+
+class CamundaProcess(models.Model):
+    __core__ = False
+
+    id = models.TextField(db_column='id_', primary_key=True)
+    rev = models.IntegerField(db_column='rev_', primary_key=True)
```
Comment (mariya) [inline src/clims/models/camunda.py]: Should primary_key be true here?

File: src/clims/api/urls.py [hunk 2]
```diff
@@ -34,7 +35,15 @@
         name='clims-api-0-sample-details'
         ),
 
-    # USER TASKS
+    # Workflow
+    url(r'^organizations/(?P<organization_slug>[^\/]+)/workflow/aggregate/task/$',
```
Comment (mariya) [inline src/clims/api/urls.py:41]: Move this to the earlier commit

**Review by mariya — CHANGES_REQUESTED** [review]
See comments

**Comment by withrocks** [comment]
I've fixed these issues now.

**Pull request merged**